src/content/nft/index.md

```diff
@@ -0,0 +1,175 @@
+
+An artist publishing work on a social network makes money for the platform who sell ads to the artists followers. They get exposure in return, but exposure doesn't pay the bills.
+
+NFTs powers a new creator economy where creators don't hand ownership of their content over to the platforms they use to publicise it. Ownership is baked into the content itself.
+
+When they sell their content, funds go directly to them. If the new owner was to go on and re-sell the NFT, the original creator can even get a % because the creator address is part of the token's metadata – metadata which can't be modified.
+
+<InfoBanner emoji=":eyes:">
+  <div>Explore, buy or create your own NFT art/collectibles...</div>
+  <ButtonLink to="/dapps?category=collectibles">Explore NFT art</ButtonLink>
+</InfoBanner>
+
+#### The copy/paste problem
+
+Naysayers often bring up the fact that NFTs "are dumb" usually alongside a picture of them screenshotting an NFT artwork. "Look, now I have that image for free!" they say smugly.
+
+Well, yes. But does googling an image of Picasso's Guernica make you the proud new owner of a multi-million dollar piece of art history?
+
+Ultimately owning the real thing is as valuable as the market makes it. The more a piece of content is screen-grabbed, shared, and generally used the more value it gains.
+
+Owning the verifiably real thing will always have more value than not.
+
+### Boosting gaming potential
+
+NFTs have seen a lot of interest from game developers. NFTs can provide records of ownership for in-game items, fuel in-game economies, and bring a host of benefits to the players.
+
+In a lot of regular games you can purchase items for you to use in your game. But if that item was an NFT you could recoup your money by selling it on when you're done with the game. You might even make a profit if that item becomes more desirable.
+
+For game developers – as issuers of the NFT – they could earn a royalty every time an item is re-sold in the open marketplace. This creates a more mutually-beneficial business model where both players and developers earn from the secondary NFT market.
+
+This also means that if a game is no longer maintained by the developers, the items you've collected remain yours under your control.
+
+Ultimately the items you grind for in-game can outlive the games themselves. Even if a game is no longer maintained, your items will always be under your control. This means in-game items become digital memorabilia and have a value outside of the game.
+
+Decentraland, a virtual reality game, lets you buy NFTs representing virtual parcels of land that you can use as you see fit.
+
+<InfoBanner emoji=":eyes:">
+  <div>Check out Ethereum games, powered by NFTs...</div>
+  <ButtonLink to="/dapps?category=gaming">Explore NFT games</ButtonLink>
+</InfoBanner>
+
+### Physical items
```

Comment:
Perhaps a comment on the future of titles/deeds to homes or cars going on the blockchain. You could buy a car/home with ETH and retrieve the deed via NFT in the same transaction in an atomic swap, and theoretically your Ethereum wallet could become the "key" to your new car or smart home (haven't seen real life examples of this yet, but seems very possible).

Reply:
Physical items of value are also being minted as NFTs to be used as collateral in defi... see below